corehq/apps/app_manager/migrations/0008_remove_uses_master_app_form_ids.py

```diff
@@ -0,0 +1,40 @@
+from django.db import migrations
+
+from corehq.apps.app_manager.models import LinkedApplication
+from corehq.dbaccessors.couchapps.all_docs import (
+    get_deleted_doc_ids_by_class,
+    get_doc_ids_by_class,
+)
+from corehq.util.couch import DocUpdate, iter_update
+from corehq.util.django_migrations import skip_on_fresh_install
+from corehq.util.log import with_progress_bar
+
+
+@skip_on_fresh_install
+def _pop_deprecated_field(apps, schema_editor):
+    app_ids = (get_doc_ids_by_class(LinkedApplication)
+               + get_deleted_doc_ids_by_class(LinkedApplication))
+    iter_update(LinkedApplication.get_db(), _pop_field, with_progress_bar(app_ids), chunksize=1)
```

Comment:
this should probably be fast (due to few docs) on all environments except prod, india and icds which may have >10k docs (haven't checked, but just thinking through how many there will be for icds). I think that might end up being pretty long to block deploy for. Are you planning to run this outside of a normal deploy for these environments?

Reply:
I had not checked, just did. There aren't that many docs: 2150 on prod, 150 on india, 300 on icds. India doesn't have many linked apps, and on icds it looks like if you go back far enough AWW builds aren't linked apps, they're regular `Application` docs. Does that seem low enough to do in a regular deploy?

I suppose the correct answer is if I don't know how long it'll take I should run this outside of deploy.

Reply:
Oh interesting I would have definitely expected more on india and icds. Seems worth running on prod and then making a call on whether or not to do it on the others

Reply:
sounds good